src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -382,7 +407,8 @@ private int GetServicePortValue(V1ServicePort servicePort, V1Container container
         private async Task<IEnumerable<EndpointInfo>> _CollectServicesToRouteAsync(
             string workloadNamespace,
             IEnumerable<V1Service> services,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken,
+            List<int> portsToIgnore)

```

Comment:
I think this has to be a map of service to ports to ignore for the service. Else if user adds port 80 to be ignore for service A, we will ignore port 80 for all services and that does not seem correct behavior.